azext_iot/digitaltwins/command_map.py

```diff
@@ -100,6 +100,7 @@ def load_digitaltwins_commands(self, _):
         cmd_group.show_command("show", "show_twin")
         cmd_group.command("update", "update_twin")
         cmd_group.command("delete", "delete_twin")
+        cmd_group.command("delete-all", "delete_all_twin", confirmation=True, supports_no_wait=True)
```

Comment:
I don't think `supports_no_wait=True` is accurate here as that implies the service is doing work async.